build.gradle

```diff
@@ -150,7 +151,7 @@ tasks.register('yarnCommon', EnigmaTask) {
 	mappings = mappingsDir
 }
 
-tasks.register('checkMappings') {
+def checkPackages = tasks.register('checkPackages') {
```

Comment:
Do we need to declare a variable here? I thought we can always just use the task name to refer to the task, or `task.getByName('checkPackages')` in the worst case